provisioning_client/tests/common_prov_e2e/prov_hsm/provisioning_hsm.c

```diff
@@ -397,6 +289,125 @@ char* iothub_hsm_get_registry_id(HSM_CLIENT_HANDLE handle)
     return result;
 }
 
+#ifdef HSM_TYPE_RIOT
+
+int hsm_client_x509_init(void)
+{
+    int result = 0;
+    initialize_device();
+    return result;
+}
+
+void hsm_client_x509_deinit(void)
+{
+}
```

Comment:
maybe add a comment here explicitly stating it's a no-op?

Reply:
Just FYI - this code is unmodified: I just moved everything together to be able to wrap it in the #define.